src/almanac/garden-lattice/garden-lattice.md

```diff
@@ -0,0 +1,32 @@
+# Garden Lattice
+
+> "To plant a garden is to believe in tomorrow."
+> \- Audrey Hepburn
+
+We enter the book through a garden lattice, an echo and forward reverberation of people who cultivate software.
```

Comment:
I'm not sure I understand what you mean by "echo and forward reverberation". What's the difference here between an echo and a reverberation? Perhaps reducing it to either "echo" or "reverberation" would be easier to understand, unless you're drawing an important distinction between them.

Reply:
Thanks for this, I think I felt that "echo" was past-tense, and "reverberation" was future-facing. Mostly I wanted to channel that this isn't a section stuck in the past. I think of a better way to state this and revise.